### DISCUSSION TITLE
v1.0.40

### DISCUSSION THREAD

**Discussion by sylikc** [sylikc/jpegview#50] [zzArchive]
## 1.0.40

- Updated to latest dcraw 9.28 (6/1/2018)
  PR #30, thanks [kakuty](https://github.com/doyaGu)

Previous version of dcraw in JPEGView was from 2008.

### Other changes:
- Created new MSI installer for JPEGView using WiX.  It's rudimentary, but functional.  See [v1.0.39.1 MSI Installer Pre-Release](https://github.com/sylikc/jpegview/releases/tag/v1.0.39.1-wix) for all the details.
- Updated Japanese translation, thanks [maboroshin](https://github.com/maboroshin) PR #46
- Updated to Turbo JPEG version 2.1.3 (2/25/2022)
- Updated to Google's WebP library version 1.2.2 HEAD at commit 7a68afa (3/21/2022)

<hr /><em>This discussion was created from the release <a href='https://github.com/sylikc/jpegview/releases/tag/'>v1.0.40</a>.</em>

**Comment by martini1234** [comment]
Please update support for animated Webp (animated GIF alternative) files. That would be great as it is still not working. Thanks.

**Reply by sylikc** [reply]
yeah unfortunately support on that is still broken at the moment.

**Reply by sylikc** [reply]
Btw @martini1234 Animated WebP is supported on the most recent release https://github.com/sylikc/jpegview/releases/tag/v1.1.41.1

**Comment by pFoxy** [comment]
There is a lack of file sorting when viewing as in Windows. So that I can change the sorting in the folder, for example, from "by name" to "by date" and when viewing the files, they were also displayed "by date".

**Reply by sylikc** [reply]
just FYI: this is related to #18 ... and that's something I'm looking into
